ledger-core/application/functest/wallet_add_amount_test.go

```diff
@@ -12,11 +12,14 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/assured-ledger/ledger-core/testutils/investigation"
 )
 
 // Creates wallet, calls /wallet/add_amount and checks it's response body.
 func TestWalletAddAmount(t *testing.T) {
-	t.Log("C4921")
+	investigation.LogCase(t, "C4921")
```

Comment:
"investigation log case" doesn't read well
May be "testrail" or "insrail" or something else pls?

Reply:
fixed